.github/workflows/schedule-nightly.yml

```diff
@@ -0,0 +1,24 @@
+name: Scheduled nightly workflows
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
+  workflow_dispatch:
+
+jobs:
+
+  run-ci:
+    name: Run nightly workflows
+    runs-on: ubuntu-latest
+    if: github.event_name != 'schedule' || github.repository == 'keycloak/keycloak-client'
```

Comment:
What is the purpose of this 'not schedule' condition?

Reply:
I've copy/pasted it from some other places :-) Like for example https://github.com/keycloak/keycloak/blob/main/.github/workflows/schedule-nightly.yml#L11 or https://github.com/keycloak/keycloak-quickstarts/blob/latest/.github/workflows/schedule.yml#L15 .

If I understand correctly, the aim of this condition is this (please correct me if I am wrong):

1) The action will executed every night in the `keycloak/keycloak-client` repository

2) The action will NOT be executed periodically in the forked repositories like `mposolda/keycloak-client` .

3) Someone in the forked repository still has a way to run this action manually in his repository if he wants to. 

I believe this condition is exactly used for that. On the other hand, running the "release nightly" in the forked repository will always fail (as some secrets are needed in GH to push the stuff into maven etc). It makes sense that someone has a way to run CI action in his repository, but "release nightly" probably does not makes sense. So maybe I can just remove the "not schedule" condition and keep just  `if: github.repository == 'keycloak/keycloak-client'` . WDYT?

Reply:
It's probably fine like this, was just wondering about the reason behind it. LGTM

Reply:
Thanks! Ok, let's keep it like that. Might be useful as it will allow to trigger more CI jobs at the same time like we're doing for the keycloak/keycloak (for keycloak-client, we will probably also have something for https://github.com/keycloak/keycloak-client/issues/87 )